lib/scroll-through-time.js

```diff
@@ -35,30 +43,42 @@ function getListener (direction) {
 export default {
 
   config: {
-    autoToggle: {
-      title: 'Auto Toggle',
-      description: 'Toggle on start.',
+    active: {
```

Comment:
I don't think you should change the name of the settings, it's breaking some behaviour (see below)